docs/repositories/issues.md

```diff
@@ -98,7 +98,13 @@ To restore an ignored issue, select **Unignore issue** from the options menu:
 
 !!! info "This section applies to GitHub repositories only"
 
-If Codacy detects code patterns that have suggested fixes, a **Fix issues** button appears above the issue list. Click this button to open a pull request including the suggested fixes and targeting the selected branch.
+If Codacy detects code patterns with suggested fixes, a **Fix issues** button appears above the issue list. Click this button to open a modal with a patch that addresses all resolvable issues. Follow the instructions in the modal to apply the patch:
+
+1.  Copy the patch content to your clipboard.
+1.  Create a new branch from your default branch.
+1.  Apply the fixes in your project's root folder, for example with the `pbpaste | patch` command.
```

Comment:
What is the "root folder"? I used "local branch" above, as it was the terminology used is the modal.
Please validate all this wording. 🙏

Reply:
It's the root directory of the repository, so the base directory of the repository (the one printed by [this command](https://stackoverflow.com/questions/957928/is-there-a-way-to-get-the-git-root-directory-in-one-command)).

Developers should be familiar with this (or it's a search away), but I can be more precise:

```suggestion
1.  Apply the fixes from your repository's root directory, e.g., with the `pbpaste | patch` command.
```